app/qml/main.qml

```diff
@@ -211,6 +211,20 @@ ApplicationWindow {
         }
     }
 
+    NotificationBanner {
+      id: notificationBanner
+
+      shouldHide: projectPanel.visible && settingsPanel.visible && stateManager.state !== "view"
```

Comment:
I can not see where this property is used or declared.

You should also probably use `||` instead of `&&`

Reply:
Sorry I forgot to remove this one.